src/gsy_e/models/strategy/heat_pump.py

```diff
@@ -134,18 +134,16 @@ def deserialize_args(constructor_args: Dict) -> Dict:
             constructor_args["order_updater_parameters"] = {
                 AvailableMarketTypes.SPOT:
                     HeatPumpOrderUpdaterParameters(
-                        update_interval=duration(
-                            minutes=constructor_args.get(
-                                "update_interval",
-                                ConstSettings.GeneralSettings.DEFAULT_UPDATE_INTERVAL)),
-                        initial_rate=constructor_args.get(
-                            "initial_buying_rate",
-                            ConstSettings.HeatPumpSettings.BUYING_RATE_RANGE.initial),
-                        final_rate=constructor_args.get(
-                            "final_buying_rate",
-                            ConstSettings.HeatPumpSettings.BUYING_RATE_RANGE.final
-                        ))
+                        update_interval=(duration(
+                            minutes=constructor_args.get("update_interval"))
+                                    if constructor_args.get("update_interval") else None),
```

Comment:
This clause is only in order to avoid `update_interval=0`, right ? Does not hurt to revert to None however I am not certain whether this is preferable to let the validator handle and reject this value. Leave it as is at any case.

Reply:
Sorry missed this, agreed of course and thanks!